windows/src/desktop/kmshell/locale/qqq/strings.xml

```diff
@@ -0,0 +1,408 @@
+﻿<?xml version="1.0"?>
```

Comment:
is `qqq` for testing? Will it need to be a custom locale added in crowdin?

Reply:
Yeah, it's an internal locale for making sure we haven't missed strings. I'm hoping we can replace it with a [pseudo locale from Crowdin](https://support.crowdin.com/pseudolocalization/), so I've left it in the app but won't upload it.